docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -231,6 +231,13 @@ test
 
 If the start page is not specified, it defaults to `about:blank`.
 
+To test webpages in local directories, you can use the `file://` protocol.
+
+```js
+fixture `MyFixture`
```

Comment:
@VasilyStrelyaev I think this question is about `testcafe` but not `testcafe-hammerhead`. I don't know how it works in `testcafe`.
@AndreyBelym ping